app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class CollectionsController < ApplicationController
+  include ModelControllerBehavior
+  self.form_class = CollectionForm
+  self.resource_class = Collection
```

Comment:
Maybe out of scope, but what do you think of making these procs/lambdas? It's basically the same thing (given that the class constructors must follow some conventions) -- but that indirection to make it a factory method seems useful.